keylime/tpm/tpm_main.py

```diff
@@ -1219,3 +1217,49 @@ def read_key_nvram(self):
             logger.debug("Invalid key length from NVRAM: %d" % (len(output)))
             return None
         return output
+
+    def stringify_pcrs(self, log: dict) -> None:
```

Comment:
This is probably just a personal preference, but this method is modifying it's arguments directly instead of returning a new struct. I'm not a fan of methods that only exist for their side effect. I know it's only called once inside of `parse_binary_bootlog`, but I think I'd be more comfortable with it if we made it a private method. 

Also, again probably nit picking, but the name makes me think it would return some string representation of all of the PCRs instead of adjusting the keys in a dict. Maybe if it's renamed to something like `_stringify_pcr_keys' that would make me less squeamish on both counts.